submissions/lalalilulu/frogger-game/app.js

```diff
@@ -0,0 +1,112 @@
+const dx = 101;
+const dy = 80;
+const collision_offset = 60;
+const player_start_x = dx * 2;
+const player_start_y = dy * 5;
+
+function calculateRandomSpeed() {
+    return Math.random() * (300 - 50) + 50;
```

Comment:
Mysterious numbers. There must be some semantic behind explaining the values.
A case for defining constants.

Reply:
These mysterious numbers for minimum and maximum enemy speed. I have created the constants for values